components/resources/brave_welcome_strings.grdp

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add descriptions - what is a title vs label vs action vs instruction / directive, etc.

Reply:
done